### PR TITLE
dev/franku/master/release notes draft

### PR DESCRIPTION
**Pull request by franku**
Please double check and add items where applicable. 

Once ready I'll go ahead and split off the bareos-18.2 release notes.

### CODE REVIEW

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 1]
```diff
@@ -24,13 +24,197 @@ This chapter concentrates on things to do when updating an existing Bareos insta
...
+* :ticket:`975`: .bvfs\_lsdirs limit offset command parameters do not work properly
+* :ticket:`1004`: Very high cpu usage on Debian stretch
+* :ticket:`1034`: Read error on tape may be misinterpreted as end-of-tape
+* :ticket:`1045`: webui login problem
+* :ticket:`1049`: Translation in German webUI
+* :ticket:`1064`: bconsole crashes after 5 minutes in restore operation due to forgotten watchdog
+* :ticket:`1073`: pthread\_detach for FreeBSD (PR169)
+* :ticket:`1091`: NDMP to NDMP Copy Job Fails
+* :ticket:`1123`: Director can crash during TwoWay Authentication
+* Fix Application Controller Plugin - CommandACLPlugin
```
Comment (joergsteffens) [inline docs/manuals/source/Appendix/ReleaseNotes.rst]: What is meant by this? Is this webui specific?

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 2]
```diff
@@ -24,13 +24,197 @@ This chapter concentrates on things to do when updating an existing Bareos insta
...
+* Fix overflowing top navigation bar content hiding tab navigation in some modules in the |webui|
+* Fix buffer overrun in function PathAppend
```
Comment (joergsteffens) [inline docs/manuals/source/Appendix/ReleaseNotes.rst]: This is also mentioned in bareos-18.2.7 release notes.

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 3]
```diff
@@ -24,13 +24,197 @@ This chapter concentrates on things to do when updating an existing Bareos insta
...
+.. list-table:: Bareos 19.2.4 Release Information
+   :header-rows: 0
+   :widths: auto
+
+   * - **Release Date**
+     - 1 December 2019
+   * - **Database Version**
+     -  2171
+   * - **URL**
+     - https://download.bareos.com/bareos/release/19.2/
```
Comment (joergsteffens) [inline docs/manuals/source/Appendix/ReleaseNotes.rst]: Will also be published on https://download.bareos.org/bareos/release/19.2/

Comment (joergsteffens) [reply docs/manuals/source/Appendix/ReleaseNotes.rst]: If https://github.com/bareos/bareos/pull/314 get merged, I've to something more to the 19.2 release notes.

File: docs/manuals/source/Appendix/ReleaseNotes.rst (hunk 1 above)
Comment (franku) [reply docs/manuals/source/Appendix/ReleaseNotes.rst]: -> webui

File: docs/manuals/source/Appendix/ReleaseNotes.rst (hunk 2 above)
Comment (franku) [reply docs/manuals/source/Appendix/ReleaseNotes.rst]: Your source was probably outdated, pls pull again.

Comment (franku) [reply docs/manuals/source/Appendix/ReleaseNotes.rst]: found it

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 4]
```diff
@@ -24,13 +24,196 @@ This chapter concentrates on things to do when updating an existing Bareos insta
...
+Changed Features
+^^^^^^^^^^^^^^^^
+* Improved Command ACL handling and error messages in the |webui|
+* Restricted console profile examples for limited and read-only purposes have been added to the |webui| package
+* Updated |webui| localization
+* Configuration: Do not allow empty passwords for mandatory passwords
+* All deamons: Generate a log message if extended attributes are disabled automatically
+* All deamons: replace crc32 implementation with a faster one
+* |dir|: Add support for MySQL 8
+
```
Comment (joergsteffens) [inline docs/manuals/source/Appendix/ReleaseNotes.rst]:
```suggestion
* PAM authentication requires a *User* configuration resource. Previously *Console* and *User* resources could be used.
* The *User* configuration resource changed.  Previously it has been an alias to the *Console* resource, which requires a password and allows some additional options. Now it only consists of a Name and ACL definitions, see :ref:`Configuration/Director.html:user-resource`.

```

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 5]
```diff
@@ -18,13 +18,201 @@ The overview about new feature of a release are shown at the :ref:`genindex` of
...
+* python-bareos: Depending on the distribution, the Python module is packaged for Python 2 and/or Python 3. Previously is has only been packaged for Python 2.
+* python-bareos: There are two variants of the Console protocol. The protocol used before Bareos-18.2 and the protocol used thereafter. The protocol since Bareos-18.2 supports TLS-PSK and PAM authentication, see :ref:`bareos-18.2.5`. Beginning with this version, python-bareos also supports both protocols. As TLS-PSK for Python (module **sslpsk**) is not available for all platform, python-bareos has integrated an automatic fallback to the old protocol.
+
+Changed Features
+^^^^^^^^^^^^^^^^
+* Improved Command ACL handling and error messages in the |webui|
+* Restricted console profile examples for limited and read-only purposes have been added to the |webui| package
+* Updated |webui| localization
+* Configuration: Do not allow empty passwords for mandatory passwords
+* All deamons: Generate a log message if extended attributes are disabled automatically
```
Comment (joergsteffens) [inline docs/manuals/source/Appendix/ReleaseNotes.rst]:
```suggestion
* All daemons: Generate a log message if extended attributes are disabled automatically
```

Is this spelling as deamon intentionally?

Comment (franku) [reply docs/manuals/source/Appendix/ReleaseNotes.rst]: typo

File: docs/manuals/source/Appendix/ReleaseNotes.rst (hunk 3 above)
Comment (franku) [reply docs/manuals/source/Appendix/ReleaseNotes.rst]: Added another line pointing to bareos.org

**Comment by arogge** [comment]
The changes have been merged outside the PR.

**Pull request closed without merging**